packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -58,7 +58,6 @@ const defaultMuiThemeOptions: ThemeOptions = {
       defaultProps: {
         PaperProps: {
           sx: theme => ({
-            borderRadius: '0px',
```

Comment:
Designs I've seen don't seem to match this, is this change requested by design or documented in Figma somewhere?

Reply:
Ah, thanks for catching! This was leftover from matching the bootstrap styling -- I'll revert in the next commit.